apps/frontend/src/components/global/groups/GroupManagement.vue

```diff
@@ -132,6 +187,63 @@ export default class GroupManagement extends Vue {
     this.editedGroup = null;
   }
 
+  displayChip(width: number, numberOfDisplayedUsers: number, members: ISlimUser[]): boolean {
+    if(window.innerWidth >= width && numberOfDisplayedUsers <= members.length) {
+      return true;
+    }
+    else {
+      return false;
+    }
+  }
+
+  displayTotalMembersChip(numberOfMembers: number): boolean {
+    // If the group has more than 3 members, than show the total chip.
+    if(numberOfMembers > 3) {
+      return true;
+    }
+    else {
+      if(window.innerWidth > 1200) {
+        return false;
+      }
+      else if(window.innerWidth >= 992 && numberOfMembers > 2) {
+        return true;
+      }
+      else if(window.innerWidth >= 0 && numberOfMembers > 1) {
+        return true;
+      }
+      else {
+        return false;
+      }
+    }
+  }
+
+  getMemberNames(users: ISlimUser[]): Object {
+    const result = [{}];
+
+    for(let i = 0; i < users.length; i++) {
+      if(!users[i].firstName && !users[i].lastName) {
+        result[i] = {identifier: users[i].email};
+      }
+      else {
+        result[i] = {identifier: `${users[i].firstName} ${users[i].lastName}`};
+      }
+    }
+    return result;
+  }
+
+  displayMembersDialog(members: ISlimUser[]) {
+    // If there is only one member, add it to a ISlimUser array.
+    if(!Array.isArray(members)) {
+      const singleMember: ISlimUser[] = [members];
+      this.selectedGroupUsers = [...singleMember];
+      this.dialogDisplayUsers = true;
+    }
+    else {
+      this.selectedGroupUsers = [...members];
+      this.dialogDisplayUsers = true;
+    }
+  }
```

Comment:
Always show all the users. You also may not need to make a copy since you aren't changing the data in the popup.